exercises/etl/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "transform": {
+    "description": "transforms the a set of scrabble data previously indexed by the tile score to a set of data indexed by the tile letter",
+    "cases": [
+      {
+        "description": "a single letter",
+        "input": {
+          "1": ["A"]
```

Comment:
I looked at several data files.  Not a single one uses `notes`.  Though that is probably a better key (long term) I'm not sure this commit should be the one to start that. So I went with `#`.

If there is some documentation that could be updated... and if you guys want (me) to build a script to update all `#` entries with `notes` to lay down the "new" convention.  I'd be happy to help.  but for now, i'm sticking with `#`